libs/src/sdk/utils/hashId.ts

```diff
@@ -15,7 +15,6 @@ export const decodeHashId = (id: string) => {
     if (isNaN(num)) return null
     return num
   } catch (e) {
-    console.error(`Failed to decode ${id}`, e)
```

Comment:
did you want to replace these with logs or not?

Reply:
No I decided against it because I didn't want to pass a logger instance into here. And these utils are mainly used in the zod schemas which already surface an error if the decoding fails

Reply:
Yeah I figured that was the reason. Sounds good to me!